pkg/kubectl/cmd/exec.go

```diff
@@ -254,7 +259,9 @@ func (p *ExecOptions) Run() error {
 
 	containerName := p.ContainerName
 	if len(containerName) == 0 {
-		glog.V(4).Infof("defaulting container name to %s", pod.Spec.Containers[0].Name)
+		if len(pod.Spec.Containers) > 1 {
+			fmt.Fprintf(p.Err, "defaulting container name to %s, use '%s describe po/%s' to see all containers in this pod.\n", pod.Spec.Containers[0].Name, p.FullCmdName, p.PodName)
```

Comment:
`describe pod/%s`? I find the letter saved in `po` more confusing than convenient